styles/ecowas_power_plant_points_gcs_wgs84_wapp_pre_delete_20230609.sld

```diff
@@ -58,12 +58,12 @@
             <se:Graphic>
               <!--Parametric SVG-->
               <se:ExternalGraphic>
-              <se:<OnlineResource xlink:type="simple" xlink:href="file:///opt/geoserver/data_dir/styles/svg/powerplants_hydro.svg"/>
+               <OnlineResource xlink:type="simple" xlink:href="file:///opt/geoserver/data_dir/styles/svg/powerplants_hydro.svg"/>
                 <se:Format>image/svg+xml</se:Format>
               </se:ExternalGraphic>
               <!--Plain SVG fallback, no parameters-->
               <se:ExternalGraphic>
-              <se:<OnlineResource xlink:type="simple" xlink:href="file:///opt/geoserver/data_dir/styles/svg/powerplants_hydro.svg"/>   
+               <OnlineResource xlink:type="simple" xlink:href="file:///opt/geoserver/data_dir/styles/svg/powerplants_hydro.svg"/>   
```

Comment:
Can you change the path from `file:///opt/geoserver/data/` to `file:///geoserver_data/data/`

Reply:
Done